### PR TITLE
Swap out poltergeist for selenium-webdriver and headless chrome for JS testing

### PR DESCRIPTION
**Pull request by mjgiarlo**
@samvera/hyrax-code-reviewers

### CODE REVIEW

File: spec/features/create_work_spec.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@
       click_link "Files" # switch tab
       expect(page).to have_content "Add files"
       expect(page).to have_content "Add folder"
-      # Capybara/poltergeist don't dependably upload files, so we'll stub out the results of the uploader:
+      # Capybara doesn't dependably upload files, so we'll stub out the results of the uploader:
```
Comment (jcoyne) [inline spec/features/create_work_spec.rb]: Still the case or did you not check?

File: spec/support/features/confirmation.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# spec/support/features/confirmation.rb
+# TODO: Delete this module when `#accept_confirm` supported by capybara w/ chrome
```
Comment (jcoyne) [inline spec/support/features/confirmation.rb]: Do you have an issue link you can put in here?

File: app/views/hyrax/base/_form_files.html.erb [hunk 3]
```diff
@@ -9,14 +9,14 @@
         <div class="row fileupload-buttonbar">
             <div class="col-xs-7">
                 <!-- The fileinput-button span is used to style the file input field as button -->
-                <span class="btn btn-success fileinput-button">
+                <span id="fileupload" class="btn btn-success fileinput-button">
```
Comment (jcoyne) [inline app/views/hyrax/base/_form_files.html.erb]: This identifier already exists on line 1. Ids should be unique

File: spec/support/features/confirmation.rb (hunk 2 above)
Comment (jcoyne) [inline spec/support/features/confirmation.rb]: What I meant to ask for was "Is there an upstream bug we should be tracking?"

**Comment by mjgiarlo** [comment]
@jcoyne All changes made. I will squash when merging.

**Comment by mjgiarlo** [comment]
Care to review again, @jcoyne?

**Pull request merged**